Feat: docs of Layout (#10)
dd3tech/dd360-components-docs: docs/layout/card-download.mdx

---
product: dd360-ds
title: React DownloadCard component
components: DownloadCard
---

# Download Card

    It is a download card that allows the user to download a file. 

### Import

<WindowEditor codeString="import { DownloadCard } from 'dd360-ds'" />
<WindowEditor codeString="import DownloadCard from 'dd360-ds/DownloadCard'" />

### Usage

The following 3 props are required to use the component

- cancelText: Text to be displayed in the cancel button.
- downloadText: Text displayed on the download button.
- description: Description of the content to be downloaded.

<ContainerComponentDoc>
    <DownloadCard
        cancelText="Do not remind me again"
        downloadText="Download Template"
        description="Download the excel template, fill in the necessary fields and re-upload the file to populate the table. You can also fill the table manually."
    />
</ContainerComponentDoc>

<WindowEditor codeString="import { DownloadCard } from 'dd360-ds'

<DownloadCard 
&nbsp;&nbsp;cancelText='Do not remind me again'

> &nbsp;&nbsp;downloadText='Download Template'

Reviewer: ```suggestion
&nbsp;&nbsp;downloadText="Download Template"
```

> &nbsp;&nbsp;description='Download the excel template, fill in the necessary fields and re-upload the file to populate the table. You can also fill the table manually.' 

Reviewer: ```suggestion
&nbsp;&nbsp;description="Download the excel template, fill in the necessary fields and re-upload the file to populate the table. You can also fill the table manually."
```

/>" />

#### downloadURL

URL of the file to download.

<ContainerComponentDoc>
    <DownloadCard
        cancelText="Do not remind me again"
        downloadText="Download Template"
        description="Download the excel template, fill in the necessary fields and re-upload the file to populate the table. You can also fill the table manually."

>         downloadURL='URL'

Reviewer: ```suggestion
        downloadURL="URL"
```
